backend/src/openarchiefbeheer/templates/admin/base_site.html

```diff
@@ -8,8 +8,6 @@
 {% block extrastyle %}
     {{ block.super }}
 
-    <link rel="stylesheet" type="text/css" href="{% static 'css/admin_overrides.css' %}">
```

Comment:
Mm... on my machine this file still exists but it's in .gitignore. I think we can keep the file and the link but remove it from .gitignore?